web/wwwroot/js/site.js

```diff
@@ -17,3 +17,12 @@ function checkIfSelectedValueIsTarget(target){
     }
 }
 
+function copyLinkToClipboard(linkToCopy){
+    // Copy the text passed into the function plus the rest of the url
+    let fullLink = window.location.origin+"/invite/"+linkToCopy
+    navigator.clipboard.writeText(fullLink);
+    
+    // Change the button styling to show the link has been copied
+    document.getElementById("copyBtn").innerHTML = "<i class='bi bi-check'></i> invite link copied"
+    document.getElementById("copyBtn").className = "btn btn-success my-3"
+}

```

Comment:
All the comments above would combine to something like 

```suggestion
function copyLinkToClipboard(linkToCopy) {
    // Copy the text passed into the function plus the rest of the url
    const fullLink = window.location.origin + "/invite/" + linkToCopy
    navigator.clipboard.writeText(fullLink)
        .then(() => {
                // Change the button styling to show the link has been copied
                const copyButton = document.getElementById("copyBtn")
                copyButton.innerHTML = "<i class='bi bi-check'></i> invite link copied"
                copyButton.className = "btn btn-success my-3"
            }
        );
}

```

Reply:
Cool thanks :) I'll reopen the branch and make these changes!